src/utils/Logger.ts

```diff
@@ -146,30 +151,99 @@ class Logger {
     return error
   }
 
-  getReactNativeLogsFilePath = () => {
-    return RNFS.CachesDirectoryPath + '/rn_logs.txt'
+  getReactNativeLogFilePath = () => {
+    return `${this.getReactNativeLogsDir()}/${format(new Date(), 'yyyy-MMM')}.txt`
   }
 
-  getLogs = async () => {
-    // TODO(Rossy) Does this technique of passing logs back as a string
-    // fail when the logs get too big?
+  getReactNativeLogsDir = () => {
+    return `${RNFS.CachesDirectoryPath}/rn_logs`
+  }
+
+  getLogsToAttach = async () => {
+    try {
+      const logDir = this.getReactNativeLogsDir()
+      const logFiles = await RNFS.readDir(logDir)
+      const path = Platform.OS === 'ios' ? RNFS.TemporaryDirectoryPath : RNFS.ExternalDirectoryPath
+      const toAttach = []
+      for (const file of logFiles) {
+        // Always remove the logs from the current month and re-copy
+        // But not the logs from previous months if they exist
+        if (file.name === `${format(new Date(), 'yyyy-MMM')}.txt`) {
```

Comment:
maybe i am missing something, but what happens on the first of the month, do we then just have that one day's worth of logs? just catching up on the comments now, I see Jean suggested this approach but with one file per week - i wonder if this is better, so that we can attach 2 weeks worth of logs (so this would range 8 - 14 days worth of logs)

Reply:
I've set the retention period of logs to 60 days and broken the logs into monthly logs. In worst case, August 30th, since the previous months logs would be 61 days old at that point we would have the entire month of August. In most cases we have have the current months logs and previous months logs.